### PR TITLE
EZEE-2930: Added form_before and form_after to correcctly generate data for Form Builder

### PR DESCRIPTION
**Pull request by mateuszdebinski**
| Question      | Answer
| ------------- | ---
| Tickets       | [EZEE-2930](https://jira.ez.no/browse/EZEE-2930)
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Another PR:
https://github.com/ezsystems/repository-forms/pull/334

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/views/content/content_edit/user_edit.html.twig [hunk 1]
```diff
@@ -48,3 +48,18 @@
 {% block close_button %}
     <a class="ez-content-edit-container__close" href="{{ path('_ezpublishLocation', {'locationId': user.versionInfo.contentInfo.mainLocationId}) }}"></a>
 {% endblock %}
+
+{% block form_before %}
+    {{ ezplatform_admin_ui_component_group('content-create-form-before', {
```
Comment (dew326) [inline src/bundle/Resources/views/content/content_edit/user_edit.html.twig]: This is user edit shouldn't be `content-edit-form-before`?

Comment (mateuszdebinski) [reply src/bundle/Resources/views/content/content_edit/user_edit.html.twig]: yes you are right, corrected and missing parameters added

File: src/bundle/Resources/views/content/content_edit/user_edit.html.twig [hunk 2]
```diff
@@ -48,3 +48,23 @@
...
+
+{% block form_after %}
+    {{ ezplatform_admin_ui_component_group('content-edit-form-after', {
+        'content': user,
+        'contentType': contentType,
+        'location': location,
+        'parent_location': user.versionInfo.contentInfo.mainLocationId,
+        'language': language
+    }) }}
+{% endblock %}
```
Comment (konradoboza) [inline src/bundle/Resources/views/content/content_edit/user_edit.html.twig]:
```suggestion
{% endblock %}

```

File: src/bundle/Resources/views/content/content_edit/user_create.html.twig [hunk 3]
```diff
@@ -46,3 +46,19 @@
     <a class="ez-content-edit-container__close" href="{{ path('_ezpublishLocation', {'locationId': parentGroup.content.versionInfo.contentInfo.mainLocationId}) }}"></a>
 {% endblock %}
 
+
```
Comment (konradoboza) [inline src/bundle/Resources/views/content/content_edit/user_create.html.twig]:
```suggestion
```

File: src/bundle/Resources/views/content/content_edit/user_edit.html.twig [hunk 4]
```diff
@@ -48,3 +48,23 @@
 {% block close_button %}
     <a class="ez-content-edit-container__close" href="{{ path('_ezpublishLocation', {'locationId': user.versionInfo.contentInfo.mainLocationId}) }}"></a>
 {% endblock %}
+
+{% block form_before %}
+    {{ ezplatform_admin_ui_component_group('content-edit-form-before', {
+        'content': user,
+        'contentType': contentType,
+        'location': location,
+        'parent_location': user.versionInfo.contentInfo.mainLocationId,
```
Comment (dew326) [inline src/bundle/Resources/views/content/content_edit/user_edit.html.twig]: I'm not sure if this should be `mainLocationId` I think in content create the real parentLocationId is passed.
//cc @webhdx

Comment (ViniTou) [reply src/bundle/Resources/views/content/content_edit/user_edit.html.twig]: in content edit, whole `parent_location` object is passed. Not sure how this works then.

Also, passing `mainLocationId` could cause bug with missing privilege rights to access it, if I am right. (@mikadamczyk )

File: src/bundle/Resources/views/content/content_edit/user_edit.html.twig [hunk 5]
```diff
@@ -46,5 +46,25 @@
...
-    <a class="ez-content-edit-container__close" href="{{ path('_ezpublishLocation', {'locationId': user.versionInfo.contentInfo.mainLocationId}) }}"></a>
+    <a class="ez-content-edit-container__close" href="{{ path('_ezpublishLocation', {'locationId': location.id}) }}"></a>
+{% endblock %}
+
+{% block form_before %}
+    {{ ezplatform_admin_ui_component_group('content-edit-form-before', {
+        'content': user,
+        'contentType': contentType,
+        'location': location,
+        'parent_location': parentLocation,
```
Comment (webhdx) [inline src/bundle/Resources/views/content/content_edit/user_edit.html.twig]: You probably don't need parent location in edit view. What did you add this?

**Pull request merged**